tabular/src/autogluon/tabular/models/tabular_nn/hyperparameters/parameters.py

```diff
@@ -76,6 +76,62 @@ def get_hyper_params():
     return hyper_params
 
 
+def get_quantile_fixed_params():
+    """ Parameters that currently cannot be searched during HPO """
+    params = {
+        'num_updates': 20000,  # maximum number of updates
+        'updates_wo_improve': 500,  # we terminate training if validation performance hasn't improved in the last 'updates_wo_improve' # of updates
```

Comment:
normally, these params are in terms of # of epochs rather than # of updates. Do you think there is big advantage to use # of updates instead?